test/automated.rb

```diff
@@ -0,0 +1,5 @@
+require_relative "./test_init"
+
+TestBench::CLI.(
+  exclude_file_pattern: %r{/_|sketch|(_init\.rb|_tests\.rb)\z}
+)
```

Comment:
TestBench::Run.('test/automated')

Reply:
@AlfonsoUceda Very odd. This came from the evt component skeleton.